__tests__/formats/scssMaps.test.js

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with
+ * the License. A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+var formats = require('../../lib/common/formats');
+var scss = require('node-sass');
+var _ = require('lodash');
+
+var dictionary = {
```

Comment:
I agree with you: in general, I tend to get lost whenever I look at the tests (and I worked a lot on them! :) ) because I'm never sure what should I mock inside the test, what should I import from the `__tests__/__[config|json_files|properties]` folders, and so on. I think having more consistency in how the tests are declared would improve their readability.
That said, I have seen a lot of Jest tests in which the objects/mocks are declared directly inside the tests (even at the cost of a lot of repetitions), but in our case we would have _a lot_ of repetitions! So I am not sure what is best.
Maybe one solution would be to rely on the "mock" functionalities of Jest, but I am not expert enough of Jest to have a clear idea now how to do it in this specific case. When I started the porting of the tests to Jest, the idea was to have two phases: in the first do the porting (all at once), in the second do the improvements (bit by bit, incrementally). That is something I want to look into after finishing (the first part of) the examples improvements. Is it ok if we keep the test as is, and postpone the discussion on what approach to use for later?

Reply:
Makes sense!